### PR TITLE
docs(PPDSC-2538) breadcrumbs documentation

### PR DESCRIPTION
**Pull request by RashikaNewsUK**
PPDSC-2538

**What**

1. Background - why this is needed
2. What did you do
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [X] Updated relevant documentation

**I have tested manually:**
- [X] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [X] The screen reader reads and flows through the elements as expected.
- [X] There are no new errors in the browser console coming from this PR.
- [X] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [X] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2538-breadcrumbs-documentation
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2538-breadcrumbs-documentation/storybook

**Review by JohnTParsons — APPROVED** [review]

**Review by JohnTParsons — DISMISSED** [review]

File: site/pages/components/breadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,671 @@
...
+          {
+            element: 'separator',
+            attribute: 'arialabel',
+            value: 'true',
+            description:
+              'Hides the separators between breadcrumb items so they are not announced by a screen reader or any other assistive technology.',
+          },
+          {
+            element: 'breadcrumbs',
+            attribute: 'arialabel',
```
Comment (mutebg) [inline site/pages/components/breadcrumbs.tsx:413]: this is `aria-current`

File: site/pages/components/breadcrumbs.tsx [hunk 2]
```diff
@@ -0,0 +1,671 @@
...
+              'Activates the breadcrumb item if it was not already selected.',
+          },
+        ],
+      },
+      aria: {
+        title: 'WAI-ARIA',
+        tableRows: [
+          {
+            element: 'nav',
+            attribute: 'arialabel',
```
Comment (mutebg) [inline site/pages/components/breadcrumbs.tsx:399]: i thnk the attribute is `aria-label`

File: site/pages/components/breadcrumbs.tsx [hunk 3]
```diff
@@ -0,0 +1,671 @@
...
+              description:
+                'If provided, overrides the stylePreset of the breadcrumb separators.',
+            },
+            {
+              attribute: 'separator',
+              type: 'Override<NewskitIconProps>',
+              default: 'chevron right',
+              description:
+                'If provided, this overrides the breadcrumb separator icons.',
+            },
```
Comment (mutebg) [inline site/pages/components/breadcrumbs.tsx:543]: Can you move this row up, before `separator.stylePreset` one

File: site/pages/components/breadcrumbs.tsx [hunk 4]
```diff
@@ -0,0 +1,671 @@
...
+            {
+              attribute: 'separator.paddingInline',
+              type: 'MQ<string>',
+              default: 'paddingInline: space020',
```
Comment (mutebg) [inline site/pages/components/breadcrumbs.tsx:547]:
```suggestion
              default: 'space020',
```

File: site/pages/components/breadcrumbs.tsx [hunk 5]
```diff
@@ -0,0 +1,671 @@
...
+              description:
+                'If provided, this overrides the breadcrumb item styling.',
+            },
+            {
+              attribute: 'typographyPreset',
+              type: 'MQ<string>',
+              default: [
+                'small - iconSize010',
+                'medium - iconSize020',
+                'large - iconSize030',
```
Comment (mutebg) [inline site/pages/components/breadcrumbs.tsx:655]:
```suggestion
                'small - utilityButton010',
                'medium - utilityButton020',
                'large - utilityButton030',
```

**Review by mutebg — CHANGES_REQUESTED** [review]

File: site/pages/components/breadcrumbs.tsx (hunk 5 above)
Comment (RashikaNewsUK) [reply site/pages/components/breadcrumbs.tsx]: @mutebg all the changes has been done. Please have a look on it

**Review by mutebg — APPROVED** [review]

**Review by baburay23 — DISMISSED** [review]

**Review by baburay23 — APPROVED** [review]

**Pull request merged**